backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -536,6 +537,23 @@ export class UserResolver {
       userEntity.lastName = lastName
     }
 
+    if (alias) {
+      if (alias.length < 5) {
+        throw new LogError('Given alias is too short', alias)
+      }
+      if (alias.length > 20) {
```

Comment:
Base on previous discussions about this topic and the usecase document, an alias will
- be an additional unique key, which can be used outside the community to identify an user
- a human readable unique key
- be defined as an alphanummeric string plus the character - and _ with a length between 5 and 20
- not support the character @ so it will be impossible to enter an email as an alias
- be shorter for predefinable system/community relevant identifiers
- support a blacklist to avoid capturing of predefinable strings

The basic idea is to have different (human readable and cryptical) user identifiers, which can be used crossover communities.
To find and access an user all identifiers could be used. But to store user information as a reference in business data e.g. in the receiving part of a cross-community-transaction the community-uuid + the users-uuid should be used and neither the alias nor the email. 
The currently used userid will only reference to internal users and should be migrated to the two columns community-uuid and user-uuid. 
With both uuids you can always identify an user crossover communities, even if the theoretical case exists that two users from differen communities will have the same uuid.